src/Editor/CMakeLists.txt

```diff
@@ -125,6 +125,10 @@ set_property(SOURCE ${SRCS} APPEND PROPERTY OBJECT_DEPENDS ${EMBEDDED_HEADER})
 create_directory_groups(${SRCS} ${HEADERS})
 
 add_executable(Editor ${SRCS} ${HEADERS})
-target_link_libraries(Editor Engine imgui imguizmo)
+if(UseMemTrack)
+    target_link_libraries(Editor Engine imgui imguizmo memtrack memtracknew)
```

Comment:
You can have multiple `target_link_libraries`.
So instead of duplicating lines. Just do `target_link_libraries(Editor memtrack memtracknew)`